src/MSAL.PCL/ConfidentialClientApplication.cs

```diff
@@ -47,19 +47,19 @@ public sealed class ConfidentialClientApplication : ClientApplicationBase
         /// <param name="appTokenCache"></param>
         public ConfidentialClientApplication(string clientId, string redirectUri,
             ClientCredential clientCredential, TokenCache userTokenCache, TokenCache appTokenCache)
-            : this(DefaultAuthority, clientId, redirectUri, clientCredential, userTokenCache, appTokenCache)
+            : this(clientId, DefaultAuthority, redirectUri, clientCredential, userTokenCache, appTokenCache)
         {
         }
 
         /// <summary>
         /// </summary>
-        /// <param name="authority"></param>
         /// <param name="clientId"></param>
+        /// <param name="authority"></param>
         /// <param name="redirectUri"></param>
         /// <param name="clientCredential"></param>
         /// <param name="userTokenCache"></param>
         /// <param name="appTokenCache"></param>
-        public ConfidentialClientApplication(string authority, string clientId, string redirectUri,
+        public ConfidentialClientApplication(string clientId, string authority, string redirectUri,
```

Comment:
Thanks for addressing issue #117. The `PublicClientApplication` looks better now. For this `ConfidentialClientApplication` here, would you consider putting that `authority` as the last parameter, so that the other 5 parameters show up in exact same order as line 48 and 49 above?

Other changes here look good too. I'll answer your question in #119 there.